### ISSUE TITLE
Half the font size in menu bar. Making it two lines.

### ISSUE THREAD

**Issue by webrockers** [tyirvine/Informant#20]
**What problem are you currently facing?**
The information takes a lot of space in the menu bar.

**What is your proposed solution to that problem?**
A fontsize about half the current size would it make possible, to show the information more compact.

**Comment by habeeba112** [comment]
To add to this, the font-size looks a little off compared to other fonts in the menu. Making the size smaller to align with the OS menu bar font-size could also resolve this issue.

**Comment by tyirvine** [comment]
<img width="415" alt="Screen Shot 2021-09-02 at 5 39 30 PM" src="https://user-images.githubusercontent.com/39813066/131929279-36b60d05-7f3d-42cb-bf28-69163e67ddea.png">

Wow! Good catch. I'm surprised I never noticed that before. I'll drop the size to match the operating system but I don't think I'm going to make it any smaller.

Here is a screenshot of the iStat menu ⤵︎
<img width="646" alt="Screen Shot 2021-09-02 at 5 42 39 PM" src="https://user-images.githubusercontent.com/39813066/131929512-852a004c-1609-4068-afbc-21ddd2e1f7ad.png">
I'm not a fan of how compact the information is as it makes it hard to read. I'll set up discussions though so this can be talked about more.

**Comment by webrockers** [comment]
Maybe make it optional?

I don't know though, how hard it is to forge it in code.

Thanks for the useful tool!
Stay healthy!

**Comment by tyirvine** [comment]
> Maybe make it optional?
> 
> 
> 
> I don't know though, how hard it is to forge it in code.
> 
> 
> 
> Thanks for the useful tool!
> 
> Stay healthy!

Yeah it wouldn't be too bad to build. I'll leave this open and maybe someone will add it in!

I'll fix the menu bar font though!
